airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -76,6 +76,7 @@ def _validate_and_transform(self, config: Mapping[str, Any]):
 
         if config.end_date:
             config.end_date = pendulum.instance(config.end_date)
+
```

Comment:
Should we add a validation for `account_ids` ?
Check for `uniqueness`, `length`, etc?

Reply:
Fixed